lib/app.jsx

```diff
@@ -489,7 +522,10 @@ export const App = connect( mapStateToProps, mapDispatchToProps )( React.createC
 								selectedTag={state.tag}
 								tags={state.tags}
 								onOutsideClick={this.onToolbarOutsideClick} />
-							<div className="source-list theme-color-bg theme-color-fg">
+							<div className="source-list theme-color-bg theme-color-fg" ref={ ( ref ) => {
+								if ( !ref ) return;
+								ref.style.flexBasis = sourceList.width + 'px';
```

Comment:
Ok, the problem I'm having then is that the `ref` doesn't exist when setting the `onMouseMove` listener. But attempting to trigger a re-render in the `ref` handler in order to provide the `ref` to the `onMouseMove` listener seems to trigger a `render` loop.